.github/workflows/ci.yaml

```diff
@@ -28,6 +28,10 @@ jobs:
     env:
       MIX_ENV: test
       DATABASE_URL: postgres://postgres:boilerplate@localhost/elixir_boilerplate_test
+      PORT: 4001
+      SECRET_KEY_BASE: G0ieeRljoXGzSDPRrYc2q4ADyNHCwxNOkw7YpPNMa+JgP9iGgJKT4K96Bw/Mf/pd
+      SESSION_KEY: test
+      SESSION_SIGNING_SALT: qh+vmMHsOqcjKF3TSSIsghwt2go48m2+IQ+kMTOB3BrSysSr7D4a21uAtt4yp4wn
```

Comment:
Is there a way to avoid doing this? Declaring them in `config/test.exs` is so much simpler 😄

Reply:
@remi Could you look at the new approach using `.env.test` and a GitHub Action _step_ to load the dotenv file to `$GITHUB_ENV`

Reply:
Could you explain _why_ we cannot use `config/test.exs` anymore? Is it because it’s not executed _at runtime_ when running `mix test` like before?

Reply:
It's a matter of compile-time vs. runtime execution in the lifecycle.

> This new configuration file was considered a very useful addition to releases. Therefore, we are also introducing `config/runtime.exs`, which is executed **after** the code compilation on all environments (dev, test, and prod) - for both Mix and releases. Our goal is to provide a better runtime configuration experience to developers, in contrast to our current configuration system which has been mostly compile-time centric.

Reply:
Right, so we _need_ to have these variables in the environment.

I think your solution works fine! `.env.test` can be used by both CI workflows _and_ by developers  running test in their local enviroment 👍